README.md

```diff
@@ -1,12 +1,14 @@
-# HiP-MicroServiceTemplate
+ # HiP-MicroServiceTemplate
 A custom "dotnet new"-template for creating backend services that make use of event sourcing.
 
 ## How to Use
 1. Clone the repository or download as ZIP-file
 1. Install the template: `dotnet new --install /path/to/downloaded/repo/Template`
-1. Create a new solution using the template: `dotnet new hipapi [--MakeSdk] -o MyNewService`
+1. Create a new solution using the template: `dotnet new hipapi [--MakeSdk] [--npmPackageName name] -o MyNewService`
```

Comment:
Better: `[--MakeSdk --npmPackageName myNewService]`
* makes it clear that NPM package name should match the project name
* makes it clear that both switches should be used together